app/controllers/users_controller.rb

```diff
@@ -0,0 +1,54 @@
+class UsersController < ApplicationController
```

Comment:
@MuhammadUsama1204 Why you are using 4 tab spaces in this controller?
Always use 2 tab spaces.